manifests/bootstrap-pod-v2.yaml

```diff
@@ -17,10 +17,10 @@ spec:
     - "--pull-secret=/etc/mcc/bootstrap/machineconfigcontroller-pull-secret"
     resources:
       limits:
-        memory: 50Mi
+        memory: 200Mi
```

Comment:
No objections, but I wonder whether it's really worth us shipping a limit at all?  I am not sure that hard limits are common in other operators.  At a random glance, e.g. the CVO only sets a `requests` but no `limits`.

Looking at the MAPI, it's the same thing.

So why not be nice to our future selves and just drop the `limits` so we don't have to bump it again?

Reply:
I asked about this internally and was pointed to
https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#resources-and-limits
which looks excellent.

Reply:
Thanks for the resource, that makes total sense. Since this is an on-going backport to 4.10, would it make sense to open a card to examine all of the current MCO resource limits(and remove where necessary)?  I'd guess a lot of them where arbitrary/legacy.

Reply:
Yeah, we don't want components critical to the cluster to be killed off if they take more resources than their limits. So it's OK to have a request, but generally we don't want limits.

Reply:
Made a new story for this so we keep track: https://issues.redhat.com/browse/MCO-765